src/App.jsx

```diff
@@ -1,104 +1,27 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
-
-export const App = () => (
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+const combinePosts = postsFromServer.map(post => {
+  const user = usersFromServer.find(person => person.id === post.userId);
+  const postComments = commentsFromServer.filter(
+    coments => coments.postId === post.id,
+  );
+
+  return {
+    ...post,
+    user,
+    comments: postComments,
+  };
+});
```

Comment:
Also let's improve the logic of this part.

Maps `postsFromServer` to create a new `combinePosts` array, each element of which contains all the necessary post information (user and comments).

```suggestion
  const combinePosts = postsFromServer.map(post => ({
    ...post,
    user: getUserById(post.userId),
    comments: getCommentsByPostId(post.id),
  }));
```